src/renderer/components/+workloads-pods/pod-details-list.tsx

```diff
@@ -30,26 +33,25 @@ enum sortBy {
 
 export interface PodDetailsListProps {
   pods: Pod[];
-  owner: KubeObject;
   maxCpu?: number;
   maxMemory?: number;
 }
 
-@observer
-export class PodDetailsList extends React.Component<PodDetailsListProps> {
-  constructor(props: PodDetailsListProps) {
-    super(props);
-    autoBind(this);
-  }
+interface Dependencies {
+  getDetailsUrl: GetDetailsUrl;
+  podStore: PodStore;
+}
 
+@observer
+export class NonInjectedPodDetailsList extends React.Component<PodDetailsListProps & Dependencies> {
   private metricsWatcher = interval(120, () => {
-    podStore.loadKubeMetrics(this.props.owner.getNs());
+    this.props.podStore.loadKubeMetrics(this.props.pods[0]?.getNs());
```

Comment:
`this.props.owner` and `this.props.pods[0]` are always the same or why this  change is done?